include/crm/common/roles.h

```diff
@@ -20,6 +20,10 @@ extern "C" {
  * \ingroup core
  */
 
+// String equivalents of enum rsc_role_e
+
+#define PCMK_ROLE_STOPPED       "Stopped"
```

Comment:
I left these internal since external code can use `pcmk_role_text(v)` to get them. However they can be used in the CIB, so I suppose this makes sense.